pkgs/servers/misc/podgrab/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "podgrab";
+  version = "2021-04-14"; # No official version tags
```

Comment:
Per https://nixos.org/manual/nixpkgs/stable/#sec-package-naming, the version string should be "unstable-2021-04-14"